rpm/container-selinux.spec

```diff
@@ -86,6 +86,7 @@ make
 # install policy modules
 %_format MODULES $x.pp.bz2
 %{__make} DATADIR=%{buildroot}%{_datadir} install install.udica-templates
+%{__make} DATADIR=%{buildroot}%{_datadir} install install.selinux-user
```

Comment:
did you mean to remove `install.udica-templates` ?

Reply:
@rhatdan Lines 88 and 89 could be squashed into a single line by appending `install.selinux-user` to the end of the previous